bidwire/notifiers/bid_notifier.py

```diff
@@ -0,0 +1,25 @@
+import logging
+import os
+import sendgrid
+import bidwire_settings
+from .base_notifier import BaseNotifier
+from sendgrid.helpers.mail import *
+from yattag import Doc
+
+log = logging.getLogger(__name__)
+
+ITEMS_DELIMITER = " ### "
+
+
+class BidNotifier(BaseNotifier):
```

Comment:
I would like us to avoid deepening this class hierarchy beyond the two levels we already have -- it adds levels of indirection (and, thus, complexity) that can be avoided. If there is code duplication you are trying to avoid here, most (all?) of it can be done via composition.

As I see it, there are two main things that are shared between all the notifiers: the fact that an email is sent, and the fact that an HTML body for the email has to be created. Even though there is some commonality between how the email bodies look like (they are all a bulleted list of new items), I am not sure it is worth extracting those pieces themselves into shared code (I recognize we've already started down this path; what I'm saying is let's not overdo it further 😃 ). We could have built all of this without needing to build a hierarchy of notifiers at all (e.g. there could be a Notifier that takes a function that makes an email body and some recipients, and separate functions that make each email type of body).

Given that we already have a hierarchy, and it is out of scope to eliminate that in this PR, let's go for the minimum changes that make it possible to have a Massgov notifier that inherits from the existing BaseNotifier to send its email. You might need to move some pieces of building the email body into the existing CommBuys and City of Boston notifiers. If that feels like a lot of duplication, you can extract some of it into a shared library.

(I recognize that this comment has ended up being long-winded and possibly not fully clear. Shout if this doesn't seem to make sense. Also happy to discuss in mostly-real-time in the Slack.)